src/index.html

```diff
@@ -11,6 +11,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form"
+      action="http://localhost:8080"
+      method="get"
+      data-qa="big"
+    >
+      <label>
+        <img
+          src="./images/Search.svg"
+          alt="Search icon"
+          class="icon icon--big"
+        >
+        <input
+          type="text"
+          class="search_bar big"
```

Comment:
checklist point 2
write all the styles for search_bar, but for .search_bar--small write only those styles that differ in design